spec/spec_helper.rb

```diff
@@ -14,9 +14,26 @@
   config.default_cassette_options = {:record => :none, :allow_unused_http_interactions => true}
   config.hook_into :webmock
 
-  Rails.application.secrets.autosde_defaults.keys.each do |secret|
-    config.define_cassette_placeholder(Rails.application.secrets.autosde_defaults[secret]) do
-      Rails.application.secrets.autosde[secret]
+  defaults = {
+    "appliance_host"        => "autosde-appliance-host",
+    "site_manager_user"     => "autosde",
+    "site_manager_password" => "change_me"
```

Comment:
If you didn't want to "duplicate" the values you could always
```suggestion
    "appliance_host"        => credentials_autosde_host,
    "site_manager_user"     => credentials_autosde_user,
    "site_manager_password" => credentials_autosde_password
```

Reply:
Yeah, I guess I wanted to see what the usage was down below to better understand if i could eliminate it entirely and do it inline.